phoenix-core/src/main/java/org/apache/phoenix/coprocessor/TaskRegionObserver.java

```diff
@@ -199,6 +200,7 @@ public void run() {
                             Task.addTask(connForTask, taskRecord.getTaskType(), taskRecord.getTenantId(), taskRecord.getSchemaName(),
                                     taskRecord.getTableName(), PTable.TaskStatus.STARTED.toString(),
                                     taskRecord.getData(), taskRecord.getPriority(), taskRecord.getTimeStamp(), null, true);
+                            connForTask.commit();
```

Comment:
@gokceni why do we need to commit here?

Reply:
removed

Reply:
@gokceni looks like `connForTask.commit()` is still there.